### ISSUE TITLE
Query should support `contains` operator

### ISSUE THREAD

**Issue by ahmadsholehin** [deepstreamIO/deepstream.io-provider-search-rethinkdb#87]
Example data model:
A project can have many collaborators, thus `userIds` are stored as an array in the `collaborators` field.
A user needs to query for projects that the user owns, as well projects that the user is a collaborator of.

Ideally, there should be a `contains` operator for a query as such:
```
{
  "table": "projects",
  "query": [
    [ "collaborators", "contains", "<userId>" ]
  ]
}
```

Any way this can be included?

**Comment by yasserf** [comment]
Yup, the query parser class has a function which accepts mappings. Feel free to raise a PR and I’ll be happy to merge with a passing test!
